spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-local/src/main/java/org/springframework/cloud/data/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -57,10 +57,17 @@ public LocalModuleDeployer(ModuleLauncher launcher) {
 	@Override
 	public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		String module = request.getCoordinates().toString();
+		if (request.getCount() != 1) {
+			logger.warn(String.format("%s only supports a single instance per module; ignoring count=%d for %s",
```

Comment:
There wouldn't be port conflicts since ports are allocated. This was primarily to avoid the refactoring that would be necessary for launching multiple instance per module here *at least for now*, given that in XD 1.0 we never ran more than one module instance in the same process. We can definitely revisit.